drafts/draft-ietf-sfc-proof-of-transit.xml

```diff
@@ -1102,8 +1109,20 @@ module ietf-pot-profile {
     </section>
 
     <section anchor="Security" title="Security Considerations">
-      <t>Different security requirements achieved by the solution approach are
-      discussed here.</t>
+      <t>POT is a mechanism that is used for verifying the path through which
+	  a packet was forwarded. The security considerations of IOAM in general
+	  are discussed in <xref target="I-D.ietf-ippm-ioam-data"/>. Specifically,
+	  it is assumed that POT is used in a confined network domain, and 
+	  therefore the potential threats that POT is intended to mitigate should
+	  be viewed accordingly. POT prevents spoofing and tampering; an attacker
+	  cannot maliciously create a bogus POT or modify a legitimate one. 
+	  Furthermore, a legitimate node that takes part in the POT protocol 
+	  cannot masquerade as another node along the path. However, POT does
+	  not provide resiliency to man-in-the-middle attacks; for example, an
+	  attacker can replay a POT, or swap the POT values between different 
+	  packets. These considerations are discussed in detail in the rest
```

Comment:
This statement is not correct. POT does secure you from MIM attacks, i.e. we'll detect replays (because POLY-3 is unique and per packet) ; Similarly, if POLY-2 is linked to the packet content (e.g. using a hash of the packet for POLY-2), you can also protect against swapping out POT fields between packets.